src/cls/_ZPM/PackageManager/Client/FilesystemServerDefinition.cls

```diff
@@ -196,9 +196,8 @@ Method AddCacheItem(pModuleFileName As %String, pSubDirectory As %String, Output
 			// Contained multiple documents - tricky! We'll just continue.
 			Quit
 		}
-		
 		Set tExt = $ZConvert($Piece($Get(tLoadedList),".",*),"U")
-		If (tExt '= "ZPM") {
+		If (tExt '= $EXTRACT(##class(%ZPM.PackageManager.Developer.Document.Module).#EXTENSION,2,)) {
```

Comment:
Now for all checks on extensions, we should check for both MODULE and ZPM right?